cmd/terraform-registry/main_test.go

```diff
@@ -26,3 +26,21 @@ func TestParseAuthTokenFile(t *testing.T) {
 	is.Equal(tokens[1], "bar")
 	is.Equal(tokens[2], "baz")
 }
+
+func TestParseAuthTokenFileJson(t *testing.T) {
```

Comment:
Can you change the name of the other test to be suffixed with `Newlines` or similar? `TestParseAuthTokenFileNewlines`

Reply:
And please capitalise `JSON` in the test name

Reply:
Updated